RELEASING/release-notes-1-3/README.md

```diff
@@ -0,0 +1,73 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes for Superset 1.3
+
+Superset 1.3 focus on hardening and polishing the superset user experience, with tons of UX improvements and bugfixes focused on charts, dashboards, and the new dashboard-native filters.
+
+- [**User Experience**](#user-experience)
+- [**PR Highlights**](#pr-highlights)
+- [**Breaking Changes and Full Changelog**](#breaking-changes-and-full-changelog)
+
+# User Experience
+One major goal of this release is to improve and harden dashboard-native filters. These filters live at the dashboard-level instead of within a chart, and affect all charts under their scope within a dashboard. Improvments in this release include clearer visual indicators of what charts are within the scope of a selected filter.
```

Comment:
```suggestion
One major goal of this release is to improve and harden dashboard-native filters. These filters live at the dashboard level instead of within a chart and affect all charts under their scope within a dashboard. Improvements in this release include clearer visual indicators of what charts are within the scope of a selected filter.
```